src/main/java/uk/gov/service/notify/SendEmailResponse.java

```diff
@@ -0,0 +1,78 @@
+package uk.gov.service.notify;
+
+import org.json.JSONObject;
+
+import java.util.Optional;
+import java.util.UUID;
+
+public class SendEmailResponse {
```

Comment:
I have avoided using an a super class since the objects are small. It's slightly a style preference when build POJOs.